Cakes-Co/index.html

```diff
@@ -1,16 +1,107 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+  </head>
+
+  <body>
+    <header class="header__style">
+      <article class="on-top">
+        <img
+          class="logo"
+          src="https://media.istockphoto.com/id/1403306417/photo/text-cake-stylized-as-a-birthday-cake-stylish-design-for-a-brand-label-or-advertisement-3d.jpg?s=612x612&w=0&k=20&c=vaFJo5V7kbjGB3jcwKATUsBsKsFfPdiSMK54sUr2BUI="
+          alt="it is used for website"
+        />
+        <p class="logo__text">The best cakes in town delivered your door!</p>
+      </article>
+    </header>
+
+    <nav id="hamnav">
+      <label for="hamburger">&#9776;</label>
+      <input type="checkbox" id="hamburger" />
+
+      <div id="hamitems">
+        <a class="nav__link" href="#">Home</a>
+        <a class="nav__link" href="#">Cakes</a>
+        <a class="nav__link" href="#">Ordering</a>
+        <a class="nav__link" href="#">Lessons</a>
+        <a class="nav__link" href="#">About</a>
+      </div>
+    </nav>
```

Comment:
I'm glad to see that you've successfully resolved the issue with the hamburger menu.